plans/e2e/ffi.fmf

```diff
@@ -20,7 +20,7 @@ adjust:
         how: shell
         script: |
            cd tests/e2e
-           ./set-ffi-env-e2e --qm-setup-from-gh-url=https://raw.githubusercontent.com/containers
+           ./set-ffi-env-e2e --branch-qm=fix-qm-tmp-dir  --qm-setup-from-gh-url=https://raw.githubusercontent.com/Yarboa
```

Comment:
Hey Yariv, https://raw.githubusercontent.com/Yarboa is it intentional ?

Reply:
Yes, once PR is merged it will use the default main qm branch

Reply:
@dougsland 
Please refer this with customized setup it was passing https://artifacts.dev.testing-farm.io/b32d9e29-73c3-496b-aed9-ead1572f831a/

This test with non merged setup will fail ffi tests